README.md

```diff
@@ -101,11 +118,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[x] `$(".post").html()`
+[x] `document.getElementsByClassName("post")[0].innerHTML`
+[x] `document.getElementsByClassName("post").innerHTML`
```

Comment:
This last would work in that you wouldn't error out but you'd always get undefined. `document.getElementsByClassName(..)` returns an array-like object which doesn't have an innerHTML attribute.